README.md

```diff
@@ -56,77 +56,33 @@ make backend
    See the [Official installation instructions](https://istio.io/latest/docs/setup/getting-started/#install)
    for more information.
 
-2. Clone the apiclarity repository to your local system
+2. Add Helm repo
 
    ```shell
-   git clone https://github.com/apiclarity/apiclarity
-   cd apiclarity
+   helm repo add apiclarity https://apiclarity.github.io/apiclarity
    ```
 
-3. Deploy APIClarity in K8s. It will be deployed in a new namespace `apiclarity`:
+3. Deploy APIClarity with Helm
```

Comment:
please update also `Testing with a demo application` to use helm command

Reply:
Ok